models/patientModel.js

```diff
@@ -14,10 +14,26 @@ const patientsSchema = new mongoose.Schema(
             type: String,
             required: true,
         },
-    },
-    {
-        timestamps: true,
-    }
+        isDoctor: {
```

Comment:
we need psychiatrist as a different entity

Reply:
registering different like the patients?

Reply:
Yes, that way.. we can have an admin.